packages/storybook-html/src/components/core/date.story.mdx

```diff
@@ -28,18 +28,19 @@ These date formats are designed for
     <tbody>
         <tr>
           <td>
-          10:00 AM <br />
-          2:30 PM
+          10:00AM <br />
```

Comment:
Based on the UQ standards link below 
https://marketing-communication.uq.edu.au/written-style-guide
it is seen as am/pm rather than AM/PM